### ISSUE TITLE
Is Scoped Registry Install still available

### ISSUE THREAD

**Issue by JulioW33** [ValveSoftware/unity-xr-plugin#96]
Hi,
I worked on a project with the openVR XR plugin, and it was installed from a scoped registry. This project runs on Unity 2020.1.8. The new project I am working on runs on Unity 2020.3.4, and I can't manage to install the plugin using a scope registry. 
I can still install the plugin from source but then the plugin path is local.
![Screenshot_31](https://user-images.githubusercontent.com/37349169/119106889-08d71d00-ba1f-11eb-9fdf-5e743d229dcd.png)
So my question is can I still use the scope registry or do I need to download it from source, at least for now?
Thank you for your answers.

**Comment by zite** [comment]
Yeah, we no longer use the scoped registry. You’ll need to download the package to use it in projects.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Jules Paradeis ***@***.***>
Sent: Friday, May 21, 2021 4:29:11 AM
To: ValveSoftware/unity-xr-plugin ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [External Mail] [ValveSoftware/unity-xr-plugin] Is Scoped Registry Install still available (#96)


Hi,
I worked on a project with the openVR XR plugin, and it was installed from a scoped registry. This project runs on Unity 2020.1.8. The new project I am working on runs on Unity 2020.3.4, and I can't manage to install the plugin using a scope registry.
I can still install the plugin from source but then the plugin path is local.
[Screenshot_31]<https://user-images.githubusercontent.com/37349169/119106889-08d71d00-ba1f-11eb-9fdf-5e743d229dcd.png>
So my question is can I still use the scope registry or do I need to download it from source, at least for now?
Thank you for your answers.

**Comment by ripandy** [comment]
Hi,
Are there any known issues when adding package directly from github (see below)?
As It seems I have successfully imported this way (last image).
Just wondering since this is not written on installation anywhere, is there any problem?
Note : requires Unity 2019.4 or later

![image](https://user-images.githubusercontent.com/1290720/122328232-3f468000-cf6a-11eb-9555-d41df7bffa9f.png)

![image](https://user-images.githubusercontent.com/1290720/122328580-e4f9ef00-cf6a-11eb-8802-7371045898d0.png)

https://github.com/ValveSoftware/unity-xr-plugin.git?path=com.valve.openvr#v1.1.4

![image](https://user-images.githubusercontent.com/1290720/122328645-fe9b3680-cf6a-11eb-9cdc-6b014fede1f3.png)
